pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -36,13 +38,19 @@ import (
 type v1beta1PolicyApplier struct {
 	jwtPolicies []*model.Config
 	// TODO: add mTLS configs.
-	// TODO: add v1alpha1 fallback configs.
 
 	// processedJwtRules is the consolidate JWT rules from all jwtPolicies.
 	processedJwtRules []*v1beta1.JWT
+
+	alphaApplier authn.PolicyApplier
 }
 
 func (a *v1beta1PolicyApplier) JwtFilter(isXDSMarshalingToAnyEnabled bool) *http_conn.HttpFilter {
+	if len(a.processedJwtRules) == 0 {
+		log.Infof("RequestAuthentication (beta policy) not found, fallback to alpha if available")
```

Comment:
change the level to debug or something.

if we enable the beta policy by default this will be causing lots of log spamming because all customer don't have beta policy

Reply:
Good catch. Change to debug.